pkg/interface/src/views/apps/publish/components/EditPost.tsx

```diff
@@ -37,7 +37,7 @@ export function EditPost(props: EditPostProps & RouteComponentProps) {
       await api.graph.addNodes(ship, book, nodes);
       await waiter(p => {
         const [rev] = getLatestRevision(p.note);
-        return rev === newRev;
+        return rev.valueOf() === newRev;
```

Comment:
`getLatestRevision` in `lib/publish.ts` should be changed such that `return [revNum.toJSNumber(), title.text, body.text, rev.post]`; is the return value.